dbt-tests-adapter/pyproject.toml

```diff
@@ -52,6 +52,16 @@ include = ["dbt/tests", "dbt/__init__.py"]
 [tool.hatch.build.targets.wheel]
 include = ["dbt/tests", "dbt/__init__.py"]
 
+[tool.hatch.envs.default]
```

Comment:
I'm surprised we didn't have this before. This allows us to develop within `dbt-tests-adapter` using a hatch environment (to get things like autocomplete in the IDE and whatnot).

As an aside, the `@ {root:uri}/..` bit is what local dev will look like in the monorepo.  Branches will no longer need to be set for packages in the `dbt-adapters` repo since `git` will pull all of them down at the same time. We'll still need to specify branches for `dbt-common` and `dbt-core` since they'll be outside of the repo.

Reply:
Would this impact individual adapters installing a particular version of core/other deps?

Reply:
This affects the development environment of `dbt-tests-adapter` itself. `hatch` environments are akin to `dev-requirements.txt`; they do not affect what gets packaged and shipped.